MoTrPAC/motrpac-frontend
# Connects to #89. Adding QC sample counts to dashboard visualization.

### Key Changes

#### Note: QC sample counts are labeled as "Reference Standards" per Archana's suggestion

1. Adding QC sample counts to the highlighted totals (e.g. genomics, metabolomics, proteomics) at the top of dashboard page.
2. Adding QC sample counts to dashboard bar plot.
3. Adding option to show/hide QC sample counts in dashboard sample count table.
4. Added doughnut charts to visualize total QC sample counts.

#### Steps to test:
1. Sign into local instance of this branch and go to ^/dashboard.
2. Expect to see the totals of _genomic_ and _metabolomics_ assays are great than those currently seen in production.
3. Expect to see the bar plot showing stacked _study assays_ and _reference standards_.
4. Expect to see a _checkbox_ option to show/hide the _reference standards_ (QC samples counts) in the table.
5. Expect to see 2 new doughnut charts to the right of the page showing the total _reference standards_ for both internal and external releases.
